clang/lib/Parse/ParseDeclCXX.cpp

```diff
@@ -480,10 +468,7 @@ Decl *Parser::ParseExportDeclaration() {
 
   while (!tryParseMisplacedModuleImport() && Tok.isNot(tok::r_brace) &&
          Tok.isNot(tok::eof)) {
-    ParsedAttributes DeclAttrs(AttrFactory);
-    MaybeParseCXX11Attributes(DeclAttrs);
```

Comment:
I wonder if we can private-ize/hide `MaybeParseXXAttributes` so that only that function can call it?

Reply:
Probably not, some place still use it, and we aren't able to replace it with the new function due to branching depending on different tokens. For example see ParseDeclCXX.cpp in `ParseLinkage`